waffle-compiler/src/findInputs.ts

```diff
@@ -12,7 +12,7 @@ export function findInputs(sourcePath: string) {
       const filePath = path.join(dir, file);
       if (isDirectory(filePath)) {
         stack.push(filePath);
-      } else if (file.endsWith('.sol')) {
+      } else if (file.endsWith('.sol') || file.endsWith('.vy')) {
```

Comment:
I think I'd prefer having the extension passed as an argument to findInputs. That way we won't have weird errors where we accidentally mix languages. So having `Main.vy` and `Dependency.vy` as well as `Main.sol` and `Dependency.sol` in the same directory wouldn't interfere.

Reply:
Also this is untested, because there are no e2e tests where you have vyper code with imports